framework/error.go

```diff
@@ -1,4 +1,4 @@
-package operator
+package framework
```

Comment:
I renamed the `operator` package to `framework` since this matches our working when we talk about it. I think it is more appropriate now.